### ISSUE TITLE
Item Piles (Trading) compatibility

### ISSUE THREAD

**Issue by kgar** [kgar/foundry-vtt-tidy-5e-sheets#151]
Related to the same issue in "Let's Trade 5e", Item Piles has functionality that allows trading items between players by right-clicking on an item in the inventory.
This image is what should appear on right click (and does with old Tidy Sheet). The option does not appear with Tidy Alpha.

![image](https://github.com/kgar/foundry-vtt-tidy-5e-sheets/assets/4632129/34d0334e-ba10-468c-915a-413a8a248bee)

Item Piles has a great deal of other functionality, but this feature replicates the trading feature of other mods like "Let's Trade 5e".

**Comment by kgar** [comment]
Tentatively,
```js
Hooks.once('tidy5e-sheet.ready', (api) => {
  api.registerActorItemContextMenuOption(
    new api.models.ContextMenuOption({
      iconCssClass: 'fas fa-user',
      text: 'MyLocalizationKey.Here',
      onClick: (item) => onGiveToCharacterClicked(item),
      enabled: (item) => shouldIShowThisOption(item) // 👈 for example, `item.type !== 'spell'` will prevent this from showing for spells
    })
  );

  // etc., any other registrations
});
```

**Comment by kgar** [comment]
Taking a step back, I think it's better to just use the built-in hook.
This is an area where, if I really want to implement my own context menu later, I can campaign for compatibility at that time, if additional compatibility efforts are needed.

**Comment by kgar** [comment]
Closing as completed. Compatibility released in 0.2.2
